config/environments/development.rb

```diff
@@ -9,26 +9,49 @@
   # Do not eager load code on boot.
   config.eager_load = false
 
-  # Show full error reports and disable caching.
-  config.consider_all_requests_local       = true
-  config.action_controller.perform_caching = false
+  # Show full error reports.
+  config.consider_all_requests_local = true
+
+  # Enable/disable caching. By default caching is disabled.
+  if Rails.root.join('tmp/caching-dev.txt').exist?
```

Comment:
Wow this is horrible. http://blog.bigbinary.com/2016/01/25/caching-in-development-environment-in-rails5.html